### PR TITLE
Jp 2928- update regression test using MIRI mrs ifu flight data

### PR DESCRIPTION
**Pull request by jemorrison**
Partially resolves [JP-2928](https://jira.stsci.edu/browse/JP-2928). This ticket updates the regression tests and flight data for MIRI MRS tests. 

**Checklist for maintainers**
- [x] added entry in `CHANGES.rst` within the relevant release section
- [x] updated or added relevant tests
- [ ] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [ ] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -58,6 +58,13 @@ resample
   a bug due to which parts of input images may not be present in the output
   resampled image under certain circumstances. [#7460]
 
+regtest
```
Comment (hbushouse) [inline CHANGES.rst:61]: this needs to go before resample, in order to maintain alphabetic order.

**Comment by hbushouse** [comment]
@jemorrison  This just needs some simple updates to finalize. Can that be done soon?

**Comment by jemorrison** [comment]
It has been a while for this. I will rerun this on regression tests to make sure all the data is on artifactory
And it is ready to be merged

File: CHANGES.rst (hunk 1 above)
Comment (hbushouse) [reply CHANGES.rst:153]: Still need to fix the placement of this log entry

Comment (hbushouse) [reply CHANGES.rst:153]: D'Oh! Never mind. Wow, my brain is fried (i does come after a ...)

File: jwst/regtest/test_miri_mrs_extract1d.py [hunk 2]
```diff
@@ -8,14 +10,18 @@
 @pytest.mark.bigdata
 def test_miri_mrs_extract1d_nominal(rtdata, fitsdiff_default_kwargs):
     """Test running extract_1d on an s3d cube containing a point source"""
-    rtdata.get_data("miri/mrs/miri_003_det_image_seq1_MIRIFUSHORT_12SHORTexp1_s3d.fits")
+    # input s3d are created using the same data that was used in
+    # test_miri_mrs_spec3_ifushort: run calwebb_spec3 on
```
[the remark this answers is not in the document]
Comment (hbushouse) [reply jwst/regtest/test_miri_mrs_extract1d.py:14]: Still need to clean up this text. I still can't parse what these 2 lines are supposed to be saying.

File: CHANGES.rst [hunk 3]
```diff
@@ -64,6 +143,18 @@ resample
...
 
+- Carry through good bits correctly for the variance array [#7515]
+
+residual_fringe
+---------------
+
+- Fix bug in Residual Fringe step handling of NaNs in input [#7471]
+
+regtest
+-------
```
Comment (hbushouse) [inline CHANGES.rst:154]: This needs to be moved up the top section, for 1.10.1 unreleased, because it wasn't part of 1.10.0.

**Comment by jemorrison** [comment]
Closing this PR because it is so old too many conflicts. Opened a new cleaner PR:#7539

**Pull request closed without merging**